benchmarks/babelstream/src/babelStreamCommon.hpp

```diff
@@ -0,0 +1,433 @@
+#pragma once
+#include <boost/tokenizer.hpp>
```

Comment:
I am not sure if this is part of boost compile time header. 
If not this will introduce additional dependencies for alpaka, at least benchmarking. 
Please remove the include and use best stl to split strings, I assume this is the reason why it is included.

Reply:
The CI is maybe passing but the reason is only that all boost packages are installed. I tried it on our development system HAL

```
spack load openpmd-api@0.15.2 %gcc@12.2.0 \
    ^adios2@2.9.2 \
    ^hdf5@1.14.3 \
    ^openmpi@4.1.5 +atomics +cuda cuda_arch=80 \
    ^python@3.11.6 \
    ^py-numpy@1.26.2
spack load boost@1.83.0 %gcc@12.2.0
```
error:
```
 fatal error: boost/tokenizer.hpp: No such file or directory
```

Reply:
I tested it, the include can be removed because it is not used

Reply:
removed yes, i found CI error in the morning and removed boost tokenizer usage; but forgot tto remove header. now removed.